protocol/contracts/beanstalk/init/InitDistribution.sol

```diff
@@ -0,0 +1,55 @@
+/*
+ SPDX-License-Identifier: MIT
+*/
+
+pragma solidity ^0.8.20;
+
+import {C} from "contracts/C.sol";
+import {AppStorage, Storage} from "contracts/beanstalk/AppStorage.sol";
+import {ShipmentPlanner} from "contracts/ecosystem/ShipmentPlanner.sol";
+
+// NOTE: Values are arbitrary placeholders.
+
+interface IBeanstalk {
+    function setShipmentRoutes(Storage.ShipmentRoute[] calldata shipmentRoutes) external;
+}
+
+/**
+ * @author funderbrker
+ * @notice Initializes logistics and shipment routes.
+ */
+contract InitDistribution {
+    address constant BEANSTALK = 0xC1E088fC1323b20BCBee9bd1B9fC9546db5624C5;
+
+    AppStorage internal s;
+    IBeanstalk beanstalk = IBeanstalk(BEANSTALK);
+
+    address shipmentPlanner = address(new ShipmentPlanner());
```

Comment:
typically we try to create vanity addresses for external contracts. Could we have a hardhat script script that deploys the shipment planner, and put the address of that as an init input?

Reply:
updated here:
https://github.com/BeanstalkFarms/Beanstalk/pull/860/commits/1c333eadf4106d25342ca7ede416bea79453891a#diff-b0451bea09abb2293197eb00e388806cf4949fc7fd2952a7c926693780a93f60L32-L35

vanity deploy script & address can be set outside this PR